### ISSUE TITLE
Question about sd_feature_extraction.ipynb

### ISSUE THREAD

**Issue by sshiningg** [lilygoli/SpotLessSplats#16]
Are you extracting Stable Diffusion features from the scaled image? Why did you use 800 for `img_size` here?
![{32FD8088-F26D-4F29-B650-B19E6F651C67}](https://github.com/user-attachments/assets/c5c764fd-6809-4016-8222-c1673b8ba34e)

Thank you for your time and assistance.

**Comment by lilygoli** [comment]
yes the images are resized to 800x800 which is in the size range where stable diffusion features are meaningful. Much higher or lower resolutions lead to random unexpected behaviour in SD.

**Comment by sshiningg** [comment]
> yes the images are resized to 800x800 which is in the size range where stable diffusion features are meaningful. Much higher or lower resolutions lead to random unexpected behaviour in SD.

I wonder how to set the size when extracting SD features for images with other resolutions, such as 1920*1080, to ensure that the SD features are meaningful?

**Comment by lilygoli** [comment]
the features can only be extracted in ~800x800 resolution otherwise it will be out of distribution for the SD network and it will not work. You can extract and save the features at 800x800 and then upsample the features in your code when using them.
